admin/containers/Post/Taxonomies/List.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable */
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import {
+    AutoSizer,
+    WindowScroller,
+    List as VirtualizedList
+} from "react-virtualized";
+import styled from "styled-components";
+
+// import Scrollbars from "./Scrollbars";
```

Comment:
@tienpham94 removing this had no effect. Not sure if this was required.

Reply:
It creates a nicer looking scrollbar actually. Remove it is good also, doesn't make much of a difference in this case.